### DISCUSSION TITLE
Label Name

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#16953] [General] [answered]
hello, can i change label name?

I want to change the name of the field

version 6.1.0

**Comment by rskrzypczak** [comment, marked as answer]
In this version it is only possible by changing/adding a translation in the language panel (_Software configuration/System tools/Language Management_)

**Reply by Akaki-1991** [reply]
> In this version it is only possible by changing/adding a translation in the language panel (_Software configuration/System tools/Language Management_)

i changed it from " vtiger_field" table.

thanks.
